README.md

```diff
@@ -53,6 +53,11 @@ Transfer learning for computer vision - [https://github.com/ividal/TransferLearn
 - https://pybit.es/pages//challenges.html
 - Concurrent Futures workshop (Parallel programming in Python) [https://github.com/teoguso/concurrent-futures-tutorial](https://github.com/teoguso/concurrent-futures-tutorial)
 
+## Books
+- [Dive Into Python 3](https://diveintopython3.problemsolving.io/)
+- [Think python 2e](https://greenteapress.com/wp/think-python-2e/)
+- [Python Practice Book] (https://anandology.com/python-practice-book/index.html)
```

Comment:
Thanks for your contribution!

you have an extra space between title and link that break the formatting